cmd/kinflate/README.md

```diff
@@ -25,14 +21,79 @@ control commits.  It encourages forking a configuration,
 customizing it, and occasionally rebasing to capture
 upgrades from the original configuration.
 
+
 ## Installation
 
+`kinflate` is written in Go and can be installed using `go get`:
+
 <!-- @installKinflate @test -->
+```shell
+go get -u k8s.io/kubectl/cmd/kinflate
 ```
-go get k8s.io/kubectl/cmd/kinflate
+
+After running the above command, `kinflate` should get installed in your `GOPATH/bin` directory.
+
+## Usage
+
+This section explains sub-commands of `kinflate` using examples. 
+
+### init 
+`kinflate init` initializes the Kubernetes Application by creating an
```

Comment:
"initializes the Kubernetes Application" doesn't make a lot of sense. Maybe like this, but I'm not sure if it's accurate:

> `kinflate init` initializes a directory with a sample Deployment manifest file

Reply:
Fixed.